README.md

```diff
@@ -10,6 +10,36 @@ This is the git repository for [openSUSE:Tools/obs-service-tar_scm](https://buil
 
 The service can be used in combination with other services like [download_files](https://github.com/openSUSE/obs-service-download_files), [extract_file](https://github.com/openSUSE/obs-service-extract_file), [recompress](https://github.com/openSUSE/obs-service-recompress) or [set_version](https://github.com/openSUSE/obs-service-set_version) e.g. within the [GIT integration](https://en.opensuse.org/openSUSE:Build_Service_Concept_SourceService#Example_2:_GIT_integration) workflow.
 
+## Archive Formats
+
+### obscpio
+The obscpio archive format is the new OBS specific archive format which used and generated by the [OBS Delta Store](http://openbuildservice.org/help/manuals/obs-reference-guide/cha.obs.architecture.html#delta_store) to store SCM changes in an incremental way. 
+It is used by the obs_scm service. A file <package>.obsinfo including useful information from your SCM system is generated, so you can access this data at buildtime.
```

Comment:
It's not 100% clear to me from this wording whether `obs_scm` *generates* obscpio files or whether it *consumes* them.  It seems to suggest that they are generated elsewhere (by the delta store) and "used by" (i.e. consumed) by `obs_scm`, but I think that would be a misinterpretation which contradicts the information on `obs_scm` below.  Any chance you could tweak the wording to clarify this?